src/com/videojs/providers/HTTPVideoProvider.as

```diff
@@ -356,6 +356,7 @@ package com.videojs.providers{
             }
             else if(_hasEnded)
             {
+                _isSeeking = true;
```

Comment:
Seems like this should be pulled up out of the branches of these conditionals entirely.